terraform/deploy/outputs.tf

```diff
@@ -5,3 +5,7 @@ output "loadbalancer" {
 output "cognito" {
   value = module.cognito.outputs
 }
+
+output "route_table" {
```

Comment:
looks like this is actually a list, so maybe should be:
```suggestion
output "route_tables" {
```

Reply:
This is the one that could be clearer, changing now